saltax/match/utils.py

```diff
@@ -698,8 +698,9 @@ def get_cut_eff(
         result_dict["all_cuts"][i] = len(selected_events_all_cut) / len(selected_events)
         result_dict["all_cuts_upper"][i] = interval.high
         result_dict["all_cuts_lower"][i] = interval.low
+
         for cut_oi in all_cut_list:
-            selected_events_cut_oi = selected_events[apply_single_cut(selected_events, cut_oi)]
+            selected_events_cut_oi = selected_events[cut_func(selected_events, cut_oi)]
             # Efficiency curves with Clopper-Pearson uncertainty estimation
             interval = binomtest(len(selected_events_cut_oi), len(selected_events)).proportion_ci()
             result_dict[cut_oi][i] = len(selected_events_cut_oi) / len(selected_events)
```

Comment:
Hi Lanqing! If we still keep the code like this, I am not sure the N-1 cut acceptance is correct.

Reply:
Great thanks for the sanity check now I wake up. Is it better now?

Reply:
Haha, the thing is that if you change in this way then it won't work for the single cut acceptance. These two need to be treated differently :)

Reply:
You are right. I buy it. Let me revert.